src/Facility.LanguageServer/FsdRequestHandler.cs

```diff
@@ -27,9 +28,9 @@ protected void SetService(Uri documentUri, ServiceInfo service)
 			m_services[documentUri] = service;
 		}
 
-		protected ServiceInfo GetService(Uri documentUri)
+		protected ServiceInfo GetService(DocumentUri documentUri)
 		{
-			return m_services.TryGetValue(documentUri, out ServiceInfo service) ? service : null;
+			return m_services.TryGetValue(documentUri.ToUri(), out ServiceInfo service) ? service : null;
```

Comment:
If `DocumentUri` would be a valid key, change the dictionary to use that. `SetService` should use the same type, i.e. the API should be consistent.